examples/basis_analysis.jl

```diff
@@ -122,9 +128,10 @@ scatter!(hm, xpoints, ypoints; marker_kwargs...)
 # a unit vector would produce the minimal participation ratio. The RB vectors from before
 # produce the following ``\mathrm{PR}``:
 
+d = size(rbvecs[1], 1)
```

Comment:
BTW isn't this the same as `dimension(rbres.basis)`

Reply:
Yes that's the same and more readable, I will change it.